### DISCUSSION TITLE
I don't understand what "Feed" is for - to make it useful it would be targeted to my repos.

### DISCUSSION THREAD

**Discussion by n13** [community/community#127885] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I don't know how or why people want a feed - but to me, it's showing me a lot of things I am not interested in, and strangely, it's showing me nothing I am actually interested in. 

I expect to see this in the feed:

- Notifications on PRs in repos I am a member of. Someone made a PR - I want to know about it. 
- Mentions in repos I am a member of
- Any other things that happen in repos I am a member of - configurable

What I currently see in the feed:
- A huge number of things that have zero relevance to me, and that I don't have time for - things other people did, other repos did, and so on. Could be I touched on too many repos over the years (now decades) at Github. But I can't follow all repos I starred, or contributed to, or filed an issue on, it's way too much. 
- Lots of updates from people I don't know and don't have time for. It's weird. 

The feed acts a bit like I am totally bored, have nothing to do, and want to see what's going on on 500+ repos and I don't know how many contributors. 

But the reality is the opposite, I am very busy, and I need a place that acts like a dashboard showing all activity on all repos I am active on - I would not mind to have to set up my watched repos manually, that would be OK, it's not that many, maybe 2 main ones and 3 or 4 secondary ones. 

I realize it's possible I am missing the point of what the feed is, or is for. Anyway in its current form it's spam to me.

**Comment by praccu** [comment]
Agreed. This is a user hostile change, which is very disappointing coming from a company that's built on serving developers.

**Comment by firasm** [comment]
Generally speaking, the "purpose" of a feed is to increase user engagement and make discovery possible/easier. I don't think this is an issue for GitHub as it's not a social media company relying on ads the same way Facebook is!

Anyway, totally agree with @n13 - luckily there's a way to turn off all the junk/noise using filters. What would be NICE is if it we could have the ability to customize what's shown there. 

Here are some examples of what would be very useful to me in a screenshot:

<img width="1314" alt="Github_feed" src="https://github.com/community/community/assets/2507459/0fc863b1-194e-407a-8a03-a80fc4f65efd">

**Comment by elchininet** [comment]
I think you missed this big discussion:

https://github.com/orgs/community/discussions/65343

It has been heavily discussed, the feedback has been enormously negative but the company decided to go ahead and ignore the community.

Take a look at the thread, many of your questions have been responded or discussed there.
